Day1-Session1-Lecture/Lecture1_Molecules.ipynb

```diff
@@ -47,7 +47,7 @@
     "\n",
     "If you want to use the example data used here, you'll also need MDAnalysisTests:\n",
     "\n",
-    "`pip isntall MDAnalysis==2.0.0b0`"
+    "`pip install MDAnalysis==2.0.0b0`"
```

Comment:
```suggestion
    "`pip install MDAnalysisTests==2.0.0b0`"
```
Based on the context of the previous line I guess the package name is a typo too? @fiona-naughton can you confirm?

Reply:
I'm just going to go with context here and say it should be MDAnalysisTests - if you can have a post-merge look @fiona-naughton that'd be useful.